ntbs-service/Models/MDRDetails.cs

```diff
@@ -31,5 +31,11 @@ public class MDRDetails : ModelBase
         [DisplayName("Country in which contact occurred")]
         public int? CountryId { get; set; }
         public virtual Country Country { get; set; }
+        public bool MDRDetailsEntered =>

```

Comment:
does this need marking with `NotMapped`?

Reply:
Getters only aren't picked up by migrations - are there any other reasons we want to mark getters with NotMapped? I think we have a bunch of cases where we are and aren't using [NotMapped] on getters so can clean them all up if wanted :)

Reply:
That makes sense - i guess they are all superfluous then!